docs/guides/notebooks/configuration/groups.rst

```diff
@@ -0,0 +1,129 @@
+############################
+User groups for file sharing
+############################
+
+Create and manage closed user groups in order to share private files in the ``/project`` directory.
+
+.. _user-group-create:
+
+Create a closed group
```

Comment:
Would strongly prefer "user-managed groups" or "self-managed groups". It's more standard terminology in archives (eg https://www.canfar.net/en/docs/group_management/ ) and anyway we have no open groups to contrast with.

Reply:
Sure! I'm changing the title to "User-managed groups" and this section header to "Create a group". I left the word "closed" in other places because it does appear in the Comanage options.